### ISSUE TITLE
[Visual Requirement-Home Page-Navigation]Luminosity ratio is less than 4.5:1 for some texts showing on the Home Page. 

### ISSUE THREAD

**Issue by v-swasha** [Microsoft/pxt#2594]
**User Experience**
If the luminosity ratio is less than 4.5:1 for all the text content then it will be difficult for the visually impaired user to see what is present on the screen.

**Test Environment**
OS: RS2 Version 1703(OS Build 15063.483)
Platform: Edge.
Tool: Color contrast Analyzer

**Repro Steps**
1. Navigate to https:/makecode.com/
2. Navigate through all controls (say computing education, about Microsoft makecode, get inspired, resources elements etc)lying on the page.
3. Launch Color Contrast Analyzer tool using WAT tool bar.
4. Verify, the luminosity ratio is greater than 4.5:1 for all the texts shwoing on Home Page.

**Actual Result**
Luminosity ratio is less than 4.5:1 for some texts showing on the Home Page like Hands-on computing education,micro:bit and Microsoft logo.

**Expected Result**
Luminosity ratio should be more than 4.5:1 for some texts showing on the Home Page like Hands-on computing education,micro:bit and Microsoft logo.

**MAS Reference**
https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={a73546c7-765f-489c-b18f-afb659fe99e6}

**Suggested fix**
1.1-Ensuring that a contrast ratio of at least 4.5:1 exists between text (and images of text) and background behind the text
1.2- If design allows , Not specify background color, not specify text color, and not using technology features that change those defaults .In this technique the author avoids having to do any contrast measures by simply not specifying the text color and not specifying the background. As a result the colors of both are completely determined by the user agent.
**Tips:-**
1.The easiest fix is to increase the brightness of light colors and decrease the brightness of darker colors.
2.Exaggerate lightness differences between foreground and background colors, and avoid using colors of similar brightness adjacent to one another, even if they differ in saturation or hue.
3.During design Use the Color Contrast Analyzer to determine whether new colors have color contrast ratio of at least 4.5:1.

**Key References for Fix**
http://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-contrast.html

**Please refer attachment for more details:**
![mas1 4 3_home page](https://user-images.githubusercontent.com/30434417/28917522-951fe348-7863-11e7-9aea-74971d4a4dec.jpg)

**Comment by microsoftsam** [comment]
Fixed in Makecode.com

**Comment by v-mesala** [comment]
Bug is verified on URL : https://makecode.com/ bug still exists .Hence reactivated the bug and added the Active tag.

**Comment by v-mesala** [comment]
Correcting the comment:
Bug is verified on URL : https://makecode.com/ bug still exists .Hence Closed the bug and added the Closed tag.

**Comment by abchatra** [comment]
@v-mesala  Comments are confusing. Are you closing the bug or reactivating?
